controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook.go

```diff
@@ -23,6 +23,7 @@ import (
 	"strings"
 
 	"github.com/coredns/corefile-migration/migration"
+	"github.com/pkg/errors"
 	kubeadmv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/types/v1beta1"
 
```

Comment:
Remove this line?

Reply:
This import is required for the `errors.Wrapf()` call. Are you suggesting I get rid off that, and just return a plain error with a useful message.